specification/keyvault/data-plane/Microsoft.KeyVault/preview/7.4-preview.1/settings.json

```diff
@@ -170,19 +169,13 @@
             "description": "The value of the pool setting."
           },
           "type": {
-            "type": {
               "enum": [
                 "boolean"
               ],
             "modelAsString": true,
             "description": "The type specifier of the value."
           }
-        },
-        "required": [
-          "name",
-          "value"
-        ],
```

Comment:
Why did you remove this? A name and value certainly seem like required properties to set the value of a named setting. This information is necessary not only to set expectations for REST API callers, but for the generated SDK clients as well.